src/Hook/useFunction.js

```diff
@@ -0,0 +1,19 @@
+import { useReducerContext } from "Context";
+
```

Comment:
suggestion: think of a better name for this hook

Reply:
will think on better naming